src/templates/ratings/add.html

```diff
@@ -7,6 +7,12 @@
           {% endif %}>
       {% if isModal %}
         <h3>{{ existingReview and _('Edit Review') or _('Leave a Review') }}</h3>
+	<p class="text-small">
+           {{ _('Please read the <a href="https://developer.mozilla.org/en-US/Marketplace/User_Review_Guidelines">Review Guidelines</a> for more details about rating apps.
+           Reviews that do not meet these guidelines may be removed by our
+           moderation team without notice.') }}
+        </p>
+        <hr/>
```

Comment:
We prefer CSS solutions rather than extra elements, for performance. So it's probably better to add a border-bottom to the ```p```